core/contracts/financial-templates/implementation/PricelessPositionManager.sol

```diff
@@ -387,6 +387,12 @@ contract PricelessPositionManager is FeePayer {
         return _getCollateral(rawTotalPositionCollateral);
     }
 
+    function _reduceSponsorPosition(
```

Comment:
This just a placeholder for now?

Reply:
This PR is in draft status and is not ready for review.